cogeo_mosaic/backends/dynamodb.py

```diff
@@ -0,0 +1,140 @@
+import functools
+import itertools
+import json
+import logging
+import os
+from decimal import Decimal
+from typing import Any, Dict, List, Optional, Tuple, Union
+
+import boto3
+import mercantile
+
+from cogeo_mosaic.backends.base import BaseBackend
+from cogeo_mosaic.backends.utils import find_quadkeys
+from cogeo_mosaic.model import MosaicJSON
+
+logger = logging.getLogger()
+logger.setLevel(logging.INFO)
+
+
+class DynamoDBBackend(BaseBackend):
+    """DynamoDB Backend Adapter"""
+
+    def __init__(
+        self,
+        table_name: str,
+        mosaic_def: Optional[Union[MosaicJSON, Dict]] = None,
+        region: str = os.getenv("AWS_REGION", "us-east-1"),
+        client: Optional[Any] = None,
```

Comment:
is `client` really `Any` ?

Reply:
I couldn't find the right class for it, because it's not the same class as the S3 one. Instead it's a `boto3.resource()`